### ISSUE TITLE
Can I change display font?

### ISSUE THREAD

**Issue by fiammanda** [olivierkes/manuskript#622]

**Comment by gedakc** [comment]
Yes the display font can be changed via the **Edit -> Settings -> Views -> Text Editor" settings.

For a quick overview see [Using Manuskript to Write a Fiction Novel](http://gedakc.users.sourceforge.net/display-doc.php?name=manuskript-novel-writing).

**Comment by fiammanda** [comment]
> Yes, the display font can be changed via the **Edit -> Settings -> Views -> Text Editor** settings.
> 
> For a quick Manuskript overview see [Using Manuskript to Write a Fiction Novel](http://gedakc.users.sourceforge.net/display-doc.php?name=manuskript-novel-writing).

I'm using the portable version on Win 10 and its not working. 

And I cannot add anything under Main/Secondary category of Characters and Plots sections. Everything goes to Minor and cannot be moved.

Also I'm wondering if any CJK word count will be supported in the future.

**Comment by gedakc** [comment]
How did you install Manuskript? Did you use the binary or run-from-source-code method?  
If binary, which filename did you download and install?

**Comment by fiammanda** [comment]
> How did you install Manuskript? Did you use the binary or run-from-source-code method?
> If binary, which filename did you download and install?

> Download manuskript-0.9.0-win32.zip, unzip it, and run manuskript/manuskript.exe.

I'm using this one https://github.com/olivierkes/manuskript/releases/download/0.9.0/manuskript-0.9.0-win32.zip.

**Comment by gedakc** [comment]
When I test using `manuskript-0.9.0-win32.zip` and run under Windows 10 I am able to change both the **Font Family** and the **Font Size** and have the change take effect in the text **Editor**.

I don't know what is different on your set up that prevents changing the font.

**Comment by expilabs** [comment]
@gedakc I'm running manuskript 0.11.0 on Pop-OS! 20.10 (basically Ubuntu derived) installed via apt package manager (no snap) and I'm having the same issue as @fiammanda: when I change the font setting via Edit -> Settings -> Views -> Text Editor -> Font, nothing changes, even after closing the running instance and opening again.
No change whatsoever with both font family and size.
I would really like to be able to use a monospace font, but the default one is not and it's really annoying when writing tables.
Please let me know if I can help to isolate the cause of this issue, I'm a software developer myself and I'm not afraid to get my hands dirty :)

**Comment by olivierkes** [comment]
@expilabs Not sure about the bug, especially if it's a packaging issue (on arch linux installed from AUR it works). Any help to isolate it is welcome.

Quick fix in the meantime: use the markdown highlighter for your tables:

> ````
> ```
> Monospace
> ```
> ````
